bindings/Sofa/src/SofaPython3/Sofa/Core/CMakeLists.txt

```diff
@@ -88,5 +87,5 @@ SP3_add_python_module(
     DESTINATION  Sofa
     SOURCES      ${SOURCE_FILES}
     HEADERS      ${HEADER_FILES}
-    DEPENDS      SofaBaseUtils SofaBaseCollision Sofa.Core Sofa.Helper Sofa.SimulationCore Sofa.DefaultType SofaBaseVisual SofaPython3::Plugin
+    DEPENDS      Sofa.Simulation.Core Sofa.Component.Collision.Response.Contact Sofa.Component.Visual SofaPython3::Plugin
```

Comment:
Should be propagated to [Bindings.SofaConfig.cmake.in](https://github.com/sofa-framework/SofaPython3/blob/master/bindings/Sofa/Bindings.SofaConfig.cmake.in)